cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/BuilderUtils.java

```diff
@@ -85,11 +85,19 @@ public static void compileWithTestsAndWrite(Path sourceRootPath, boolean offline
         List<BLangPackage> packages = compiler.build();
 
         if (skiptests) {
-            outStream.println();
-            compiler.write(packages);
+            if (packages.size() > 0) {
+                outStream.println();
+                compiler.write(packages);
+            } else {
+                outStream.println("No ballerina source files to compile");
```

Comment:
No ballerina source files found to compile

Reply:
Fixed with 45ff1fd